src/bootstrap/mk/Makefile.in

```diff
@@ -71,13 +71,9 @@ ci-subset-2:
 
 ## MingW native builders
 
-TESTS_IN_MINGW_2 := \
-	tests/ui
-
-ci-mingw-subset-1:
-	$(Q)$(CFG_SRC_DIR)/x test --stage 2 $(TESTS_IN_MINGW_2:%=--exclude %)
-ci-mingw-subset-2:
-	$(Q)$(BOOTSTRAP) test --stage 2 $(TESTS_IN_MINGW_2)
+# test both x and bootstrap entrypoints
+ci-mingw:
+	$(Q)$(CFG_SRC_DIR)/x test --stage 2 tidy && $(Q)$(BOOTSTRAP) test --stage 2 --exclude tidy
```

Comment:
Doesn't this has the same problem as https://github.com/rust-lang/rust/pull/112668#issuecomment-1596013199 ?

Reply:
...that's what I get for not looking at your previous comment :sweat_smile: